src/docs/framework-integration/angular.md

```diff
@@ -89,193 +141,292 @@ import { angularOutputTarget } from '@stencil/angular-output-target';
+See the [documentation](/docs/distribution/#distribution-options) for more information on defining custom elements using the
+`dist` output target, or [update the Angular output target](#do-i-have-to-use-the-dist-output-target) to use `dist-custom-elements`.
+
+### Link Your Packages (Optional)
 
-> If you are using a monorepo tool (Lerna, Nx), skip this section.
+:::note
+If you are using a monorepo tool (Lerna, Nx, etc.), skip this section.
+:::
 
 Before you can successfully build a local version of your Angular component library, you will need to link the Stencil package to the Angular package.
 
 From your Stencil project's directory, run the following command:
 
-```bash
+```bash npm2yarn
 # Link the working directory
 npm link
-# or if using yarn
-yarn link
 ```
 
 From your Angular component library's directory, run the following command:
 
-```bash
+```bash npm2yarn
 # Link the package name
 npm link name-of-your-stencil-package
-# or if using yarn
-yarn link name-of-your-stencil-package
 ```
 
 The name of your Stencil package should match the `name` property from the Stencil component library's `package.json`.
 
-Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the changes to the Angular component library.
+Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the
+changes to the Angular component library.
 
-> **NOTE**: As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy, however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::note
+As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy,
+however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::
 
 ## Consumer Usage
 
+### Creating a Consumer Angular App
+
+:::note
+If you already have an Angular app, skip this section.
+:::
+
+From your Angular workspace (`/packages/angular-workspace`), run the following command to generate an Angular application:
+
+```bash
+npx -p @angular/cli ng generate app my-app
+```
+
+### Consuming the Angular Wrapper Components
+
 This section covers how developers consuming your Angular component wrappers will use your package and component wrappers.
 
-If you distributed your components through a primary `NgModule`, developers can simply import that module into their implementation to use your components.
+In order to use the generated component wrappers in the Angular app, you'll first need to build your Angular component library. From the root
+of your Angular workspace (`/packages/angular-workspace`), run the following command:
+
+```bash
+npx -p @angular/cli ng build component-library
+```
```

Comment:
When I get to this stage, I get an error:

![Screenshot 2023-01-27 at 8 35 51 AM](https://user-images.githubusercontent.com/1930213/215099784-d936bd58-b61a-4318-9dc2-43409b6265e1.png)

Do I need to set this flag? Or maybe there's something I did wrong when adding `stencil-library` to my `package.json`? I just manually added it, w/o an `npm i` command, which I wouldn't think would be an issue 🤔

Reply:
The `stencil-library` needed to be a peer dep instead of just a regular dep. I updated that step accordingly